src/test/resources/application-test.yml

```diff
@@ -45,29 +46,35 @@ hmpps.sqs:
       dlqName: cpr_delius_offender_events_queue_dlq
       subscribeTopicId: domainevents
       subscribeFilter: '{"eventType":[ "probation-case.engagement.created", "OFFENDER_ADDRESS_CHANGED", "OFFENDER_DETAILS_CHANGED", "OFFENDER_ALIAS_CHANGED"] }'
+      dlqMaxReceiveCount: 1
```

Comment:
For envs this is set to 3. But I believe we dont need to retry on failures if forced as we should expect them within tests.

Reply:
I could set to 3 but would require to update tests to expect 3 log statements for message recieved?